SIMPoisson.C

```diff
@@ -226,7 +226,7 @@ bool SIMPoisson<Dim>::solveStep (TimeStep&)
   if (!this->setMode(Dim::opt.eig == 0 ? SIM::STATIC : SIM::VIBRATION))
     return false;
 
-  if (!this->initSystem(Dim::opt.solver, 1, Dim::opt.eig == 0 ? 1 : 0))
+  if (!this->initSystem(Dim::opt.solver, 1 + (Dim::opt.eig != 0), Dim::opt.eig == 0 ? 1 : 0))
```

Comment:
More readable with `Dim::opt.eig == 0 ? 1 : 2` although it will result in the same

Reply:
agreed